tools/internal_ci/linux/grpc_xds_k8s_install_test_driver.sh

```diff
@@ -269,9 +269,12 @@ kokoro_setup_python_virtual_environment() {
   eval "$(pyenv virtualenv-init -)"
   py_latest_patch="$(pyenv versions --bare --skip-aliases | grep -E "^${PYTHON_VERSION}\.[0-9]{1,2}$" | sort --version-sort | tail -n 1)"
   echo "Activating python ${py_latest_patch} virtual environment"
-  pyenv virtualenv "${py_latest_patch}" k8s_xds_test_runner
+  pyenv virtualenv --no-pip "${py_latest_patch}" k8s_xds_test_runner
   pyenv local k8s_xds_test_runner
   pyenv activate k8s_xds_test_runner
+  python -m ensurepip
```

Comment:
is this needed?

Reply:
Yes, without it `python -m pip` doesn't work:

```
(k8s_xds_test_runner_2) kbuilder@sergiitk-kokoro-test1:~$ python -m pip --version
/home/kbuilder/.pyenv/versions/k8s_xds_test_runner_2/bin/python: No module named pip
(k8s_xds_test_runner_2) kbuilder@sergiitk-kokoro-test1:~$ python -m ensurepip
Requirement already satisfied: setuptools in ./.pyenv/versions/3.6.1/envs/k8s_xds_test_runner_2/lib/python3.6/site-packages
Collecting pip
Installing collected packages: pip
Successfully installed pip-9.0.1
(k8s_xds_test_runner_2) kbuilder@sergiitk-kokoro-test1:~$ python -m pip --version
pip 9.0.1 from /home/kbuilder/.pyenv/versions/k8s_xds_test_runner_2/lib/python3.6/site-packages (python 3.6)
```

Then I can do
```
(k8s_xds_test_runner_2) kbuilder@sergiitk-kokoro-test1:~$ python -m pip install -U pip==21.0.1
Collecting pip==21.0.1
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/fe/ef/60d7ba03b5c442309ef42e7d69959f73aacccd0d86008362a681c4698e83/pip-21.0.1-py3-none-any.whl (1.5MB)
    100% |████████████████████████████████| 1.5MB 838kB/s
Installing collected packages: pip
  Found existing installation: pip 9.0.1
    Uninstalling pip-9.0.1:
      Successfully uninstalled pip-9.0.1
Successfully installed pip-21.0.1
You are using pip version 21.0.1, however version 21.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```